### PR TITLE
DEVX-2519: update dashboards, grafana, add link

### PR DESCRIPTION
**Pull request by awalther28**
@waliaabhishek @jeanlouisboudart 

The proposed changes are as follows:
- update consumer dashboard (has large disclaimer on top of dashboard that warns that these are sample thresholds)
- update producer dashboard (has large disclaimer on top of dashboard that warns that these are sample thresholds)
- upgrade grafana version --> needed because "$__rate_interval" is used in upgraded dashboards, this feature only available in newer versions of grafana
  - verified that pie chart plugin still works
  - verified that auth doesn't change
- add prometheus scrape timeInterval to grafana datasource --> needed by "$__rate_interval"
- remove unneeded graphite version
- add link to observability tutorial

### CODE REVIEW

File: jmxexporter-prometheus-grafana/docker-compose.override.yml [hunk 1]
```diff
@@ -79,7 +79,7 @@ services:
       - $MONITORING_STACK/assets/prometheus/security:/etc/kafka/secrets
 
   grafana:
-    image: grafana/grafana:7.0.6
+    image: grafana/grafana:7.3.7
```
Comment (waliaabhishek) [inline jmxexporter-prometheus-grafana/docker-compose.override.yml:82]: Any reason for not bumping it to the latest 7.4.3 version instead ?

File: jmxexporter-prometheus-grafana/assets/prometheus/grafana/provisioning/datasources/datasource.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ datasources:
   isDefault: true
   # <map> fields that will be converted to json and stored in json_data
   jsonData:
-     graphiteVersion: "1.1"
+     timeInterval: 60s
```
Comment (waliaabhishek) [inline jmxexporter-prometheus-grafana/assets/prometheus/grafana/provisioning/datasources/datasource.yml:40]: Any reason for forcing 60sec interval ?

**Review by waliaabhishek — COMMENTED** [review]
Just checked the PROM scrape settings. This should match that which it does. Can you add a comment to this line mentining it here that they should match it to the lowest scrape interval used in Prom scrape setup ?

File: jmxexporter-prometheus-grafana/assets/prometheus/grafana/provisioning/datasources/datasource.yml (hunk 2 above)
Comment (waliaabhishek) [reply jmxexporter-prometheus-grafana/assets/prometheus/grafana/provisioning/datasources/datasource.yml:40]: Just checked the PROM scrape settings. This should match that which it does. Can you add a comment to this line mentining it here that they should match it to the lowest scrape interval used in Prom scrape setup ?

Comment (awalther28) [reply jmxexporter-prometheus-grafana/assets/prometheus/grafana/provisioning/datasources/datasource.yml:41]: @waliaabhishek Looks like you were able to answer your question, that `timeInterval` is the Prometheus scrape interval. It informs how grafana does its calculation for `$__rate_interval`. It isn't the lowest scrape interval but the _typical_ scrape interval that Prometheus uses. For scrape intervals that are different than what is defined in this config, one could do something like the following but that is not applicable to this project :) 
>But what to do if you have the odd metric that has been scraped with a different scrape interval? In that case, use the Min step field to enter the special scrape interval, and everything will just work again. (Pro tip: Note that the Min interval setting in the Query options does not change the scrape interval used for the $__rate_interval calculation. You can use it to get lower-resolution graphs without changing the assumed scrape interval.) 
[[source](https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/)] 

I added a comment above this config that explains it using the same language from the [grafana docs ](https://grafana.com/docs/grafana/latest/datasources/prometheus/).

File: jmxexporter-prometheus-grafana/docker-compose.override.yml (hunk 1 above)
Comment (awalther28) [reply jmxexporter-prometheus-grafana/docker-compose.override.yml]: no particular reason, 7.3.7 was the latest at the time of starting my project. I tested with 7.4.3 and everything appears alright.

Comment (waliaabhishek) [reply jmxexporter-prometheus-grafana/docker-compose.override.yml]: Please bump it to 7.4.3

**Comment by awalther28** [comment]
@waliaabhishek I did apply the feedback you provided in this commit [e5b592a](https://github.com/confluentinc/jmx-monitoring-stacks/pull/34/commits/e5b592a02cf4af91e8acc2c744dfad4a0d6fd15f). 
@jeanlouisboudart and I were on a call a few days back where he provided some feedback as well, all which is tracked in this [internal Jira ticket](https://confluentinc.atlassian.net/browse/DEVX-2553). I won't have time this week to address those feedback items. Are they necessary to complete prior to merging this PR?

**Comment by waliaabhishek** [comment]
I will let @jeanlouisboudart confirm that.

**Comment by jeanlouisboudart** [comment]
I'm fine with current state, we can iterate later on those points.

**Review by jeanlouisboudart — APPROVED** [review]

**Comment by jeanlouisboudart** [comment]
@awalther28 I've just merged, it thanks again for your work on that !

**Pull request merged**